### DISCUSSION TITLE
Error when deploying

### DISCUSSION THREAD

**Discussion by MuhammadAhmedSuhail** [vercel/community#7457] [Help]
### Summary

I was deploying a project which works locally and on other teams but gives me the following error on my hobby team,

"Your Team encountered an unknown problem. Please reach out to our support team for details."

### Steps to Reproduce

Try to deploy a working project

**Comment by amyegan** [comment]
Thanks for reaching out, @MuhammadAhmedSuhail.  It should work for you now.

[a comment by an automated account is not shown]
